data/signin.js

```diff
@@ -0,0 +1,265 @@
+class Signin{
+
+    constructor() {
+        this.selector_type = 'dusk';
+
+        this.params = {
+            group:{
+                count: null,
+                name: null,
+            }
+        }
+
+        this.selectors = {
+            email: 'signin-email_or_username',
+            password: 'signin-password',
+            submit: 'signin-signin',
+            page_heading: 'h3',
+            home_page_heading: 'h2',
```

Comment:
@satyajeet-s001 why `home` is used `home_page_heading` ?

Reply:
updated sir!